src/audio_unit/looper.rs

```diff
@@ -6,13 +6,16 @@ use State::*;
 #[derive(Debug)]
 pub enum Message {
     Toggle,
+    EnableOverdubMode,
```

Comment:
Maybe this could be named better, eg "queue overdub"?

Reply:
I like that.

Reply:
Ok renamed to `QueueOverdub`